mantidproject/mantid
# Mantid crashes when loading data and algorithm widget is hidden

See the bug report in #17933

**To test:**

Try hiding the algorithm widget and loading some data. You should not get a crash. Now delete the workspace with the widget still hidden. Now ensure you can still show/hide the widget

Fixes #17933 .

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

Attempting the show the dock when it is hidden causes a crash. This
code is only used in one place, so we should jsut remove this to get
the behaviour we want.
